lib/oli/analytics/by_activity.ex

```diff
@@ -11,18 +11,13 @@ defmodule Oli.Analytics.ByActivity do
   def query_against_project_slug(project_slug, filtered_sections) do
     project_slug
     |> get_base_query(filtered_sections)
-    |> get_query_with_join_filter(filtered_sections)
     |> Repo.all()
   end
 
   defp get_base_query(project_slug, filtered_sections) do
     subquery =
       if filtered_sections != [] do
-        Publishing.query_unpublished_revisions_by_type_and_section(
-          project_slug,
-          "activity",
-          filtered_sections
-        )
+        DeliveryResolver.revisions_filter_by_section_ids(filtered_sections, 3)
```

Comment:
I think that here, in the second argument, we should use something like Oli.Resources.ResourceType.id_for_activity to not hardcode that id. WDYT?

NOTE: Oli.Resources.ResourceType should be an alias of our module so that we don't repeat it everywhere we have to call it.